app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb

```diff
@@ -42,17 +25,4 @@ class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::IbmCloudCredent
   def self.display_name(number = 1)
     n_('Credential (IBM Cloud)', 'Credentials (IBM Cloud)', number)
   end
-
-  def self.params_to_attributes(params)
-    attrs = super.dup
-
-    attrs[:auth_key] = attrs.delete(:auth_key) if attrs.key?(:auth_key)
-
-    if %i[classic_user classic_key].any? { |opt| attrs.key?(opt) }
-      attrs[:userid]   = attrs.delete(:classic_user) if attrs.key?(:classic_user)
-      attrs[:password] = attrs.delete(:classic_key)  if attrs.key?(:classic_key)
-    end
```

Comment:
I think this just goes away if classic user/key are moved to the other class as auth_key is required, right?

Reply:
Not sure - @agrare ?